windows-msi/msi.wxs

```diff
@@ -1423,6 +1387,9 @@
             TUN/TAP Driver Installation
         -->
         <DirectoryRef Id="PRODUCTDIR">
+            <Merge Id="OvpnDcoMergeModule" Language="1033" DiskId="1" SourceFile="!(bindpath.build)$(var.PRODUCT_PLATFORM)\ovpn-dco.msm">
```

Comment:
Why do you use Language=en_US here instead of "0" as everywhere else? Will this cause problems on systems that use a different language pack?

Reply:
It shouldn't - at least works fine on my Finnish Windows 11. But let me change to 0 to be consistent.

Reply:
Oh I had to change MSM language to 0 as well - it was 1033.